androidtv/constants.py

```diff
@@ -209,7 +209,7 @@ class DeviceEnum(IntEnum):
 CMD_RUNNING_APPS_ANDROIDTV = "ps -A | grep u0_a"
 
 #: Get the running apps for a Fire TV device
-CMD_RUNNING_APPS_FIRETV = "ps | grep u0_a"
+CMD_RUNNING_APPS_FIRETV = "ps -A | grep u0_a"
```

Comment:
A unit test is failing because this constant is the same as `CMD_RUNNING_APPS_ANDROIDTV`.

Reply:
You want me to remove the FIRETV constant and refer to the ANDROIDTV one from everywhere?

Reply:
Please consolidate the two into a single `CMD_RUNNING_APPS`.

Reply:
Ok done 👍